File: java/Car.java
public class Car extends Vehicle {
    public void displayInfo() {
        super.displayInfo();
        System.out.println("model " + model);

    private String model;

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    public Car(String brand, int year, String model) {
        super(brand, year);
        this.model = model;
    }

    public Car() {
        this("Unknown", 0, "Unknown");
    }

    @Override
    public void displayInfo() {
        super.displayInfo();
        System.out.println("model " + model);
    }

}

File: python/Car.py
class Car:

    def __init__(self,brand="Toyota",model="Camry",year=2000):
        self.__brand=brand
        self.__model=model
        self.__year=year
    def getBrand(self):
        return self.__brand
    def setBrand(self,brand):
        self.__brand=brand
    def getModel(self):
        return self.__model
    def setModel(self,model):
        self.__model=model
    def getYear(self):
        return self.__year
    def setYear(self,year):
        self.__year=year
    
car_1=Car("Honda","Civic",2006)
car_2=Car()
print(car_1.getBrand(),car_1.getModel(),car_1.getYear())
print(car_2.getBrand(),car_2.getModel(),car_2.getYear())
